cmd/test/main_test.go

```diff
@@ -216,6 +696,17 @@ func TestTestCommandFlagsValidation(t *testing.T) {
 	test_testCommand_version_flags_validation(t, ctx)
 }
 
+func TestTestCommandEmptyDir(t *testing.T) {
+	setup()
+	emptyDir := t.TempDir()
+	emptyDirPaths := filepath.Join(emptyDir, "*.yaml")
+
+	readerMock.On("FilterFiles", []string{emptyDirPaths}).Return([]string{}, nil)
+	err := Test(ctx, []string{emptyDirPaths}, &TestCommandData{K8sVersion: "1.18.0", Output: "", Policy: testingPolicy, Token: "134kh"})
+
+	assert.EqualError(t, err, "No files detected")
+	readerMock.AssertCalled(t, "FilterFiles", []string{emptyDirPaths})
```

Comment:
removed this assert, this is an implementation detail which doesn't add confidence to the test